### DISCUSSION TITLE
Is Common Voice a part of the training data?

### DISCUSSION THREAD

**Discussion by yunigma** [openai/whisper#2305] [Q&A] [answered]
Hello,
thank you very much for your project!
I have a data-related question. Is the Common Voice data, from any of its languages, used as a part of the Whisper training data?
It is important to know to estimate performance fairly. Also, for my current experiments, I am generating pseudo-labels for the Common Voice train data (EN, DE, FR, ES, CA, IT) with Whisper and I need to be sure whether this data has already been seen by the model or not.
Thank you!

**Comment by glangford** [comment, marked as answer]
See
* https://github.com/openai/whisper/discussions/1414#discussioncomment-6075565

**Reply by yunigma** [reply]
Thank you!
